modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/content/Language.properties

```diff
@@ -97,6 +97,8 @@ this-application-wants-the-following-permissions=This application wants the foll
 this-operation-cannot-be-performed-because-you-cannot-impersonate-x=This operation cannot be performed because you cannot impersonate user {0}.
 this-scope-is-no-longer-available=This scope is no longer available.
 token_introspection=Token Introspection
+trusted-application=Trusted Application
+trusted-application-help=Register a Trusted Application (only available using Authorization Code or PKCE Authorization Code) allows authentication during the authorization grant flow, so there is no manual authentication grant that the user must taken as a step.
```

Comment:
I find this help text confusing. Especially the "authentication grant" part because we are only talking about authorization with OAuth 2. Unless talking about how the user that is granting authorization is identified.

Off the top of my head, I would suggest something like "Trusted applications will not ask the end-user to give consent to the access token scope."

Reply:
I'm totally open to suggestions here, I find hard to explain this kind if features without getting into the technical stuff

Reply:
Please, see: https://github.com/liferay-appsec/liferay-portal/pull/276/commits/634d46e2ae9f98f6cd8accf0010215e57986aa3e